.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: true,
+    commonjs: true,
+    es2021: true,
+  },
+  extends: [
+    'airbnb-base',
+  ],
+  parserOptions: {
+    ecmaVersion: 'latest',
+  },
+  rules: {
+    "no-console": 0,
```

Comment:
Want to allow console messages, don't require ===, increase the line length and really import, allow await inside a loop (because we often have them for saving all the data - we SHOULD set it up to be fully parallel, but that is a separate PR)